neo4j/api.py

```diff
@@ -80,13 +80,34 @@ def custom_auth(principal, credentials, realm, scheme, **parameters):
 class Bookmark:
 
     def __init__(self, value):
-        self.value = value
+        """
+        :param value: ASCII string value
+        """
+        self.values = set()
+        self.update([value,])
 
     def __repr__(self):
-        return "<Bookmark value=%r>" % self.value
+        """
+        :return: repr string with sorted values
+        """
+        return "<Bookmark values={{{}}}>".format(", ".join(["'{}'".format(ix) for ix in sorted(self.values)]))
 
     def __bool__(self):
-        return bool(self.value)
+        return bool(self.values)
+
+    def update(self, values):
```

Comment:
The bookmark is unmutable after it is created in other languages. The methods that we want to add are aiming to provide a way for a user to get a string set out and then reconstruct a Bookmark back from the string set.

This `update` method here is more like a method to modify the current Bookmark.
For example the Java driver has the following methods:
```
    /**
     * Returns a read-only set of bookmark strings that this bookmark instance identifies.
     */
    Set<String> values();

    /**
     * Reconstruct bookmark from bookmarks string values.
     * @param values values obtained from a previous bookmark.
     * @return A bookmark.
     */
    static Bookmark from( Set<String> values )
    {
        return InternalBookmark.parse( values );
    }
```
We on purpose to make `values` to return a read-only set. And also the `from` method to be static so that no one could ever modify the bookmark values after they are created.

Reply:
I refactored the code to be a staticmethod Bookmark.create(values)